src/cvodes/cvodes.c

```diff
@@ -8461,13 +8505,14 @@ static int cvEwtSetSS(CVodeMem cv_mem, N_Vector ycur, N_Vector weight)
 
 static int cvEwtSetSV(CVodeMem cv_mem, N_Vector ycur, N_Vector weight)
 {
-  N_VAbs(ycur, cv_mem->cv_tempv);
+  SUNAssignSUNCTX(CV_SUNCTX);
+  SUNCheckCallLastErrNoRet(N_VAbs(ycur, cv_mem->cv_tempv));
   N_VLinearSum(cv_mem->cv_reltol, cv_mem->cv_tempv, ONE,
                cv_mem->cv_Vabstol, cv_mem->cv_tempv);
   if (cv_mem->cv_atolmin0) {
     if (N_VMin(cv_mem->cv_tempv) <= ZERO) return(-1);
```

Comment:
Needs a `sunrealtype minval` declared above, and then this should be changed to something like
```suggestion
    minval = SUNCheckCallLastErrNoRet(N_VMin(cv_mem->cv_tempv));
    if (minval <= ZERO) return(-1);
```